offscreen-canvas/the-offscreen-canvas/offscreencanvas.transfer.to.imagebitmap.w.html

```diff
@@ -0,0 +1,215 @@
+<!DOCTYPE html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/common/canvas-tests.js"></script>
+<link rel="help" href="https://html.spec.whatwg.org/multipage/scripting.html#the-offscreen-2d-rendering-context">
+
+<script id="myWorker" type="text/worker">
```

Comment:
Unfortunately that doesn't work.

In my first pull request when I auto-generate those tests: https://github.com/w3c/web-platform-tests/pull/4265, I actually had worker.html and worker.js file. The problem comes when the wpt-runner ran these tests:
1. When the wpt-runner run a tests end with worker.html, it automatically looks for a worker.js file with the same prefix string in the file name.
2. When the wpt-runner sees a worker.js file, it automatically construct a new .html file with only one line of code: fetch_tests_from_worker(new Worker("xxx.worker.js"));

Notice that NO.1 works fine if I apply your suggested change. But NO.2 fails in these tests ends with w.html because the worker.js cannot run separately. Then Travis will complain that it detects flacky because they have the same name, "xxx.worker.html", but one pass one fail.